README.md

```diff
@@ -151,6 +151,13 @@ or
 
     mvn -Pjenkins-651 hpi:run
 
+## Setup Wizard
+
+By default, the setup wizard (Jenkins >= 2.0) is skipped when using `hpi:run`. If you want the wizard to be enabled just run:
+
+    mvn -DenableWizard hpi:run
```

Comment:
It's done this way as the way to disable is add a property. If by default it was disabled and we have to enable it adding a property we could have gone the `-PenableWizard` way.